charts/rancher-backup/files/default-resourceset-contents/provisioningv2.yaml

```diff
@@ -0,0 +1,18 @@
+- apiVersion: "provisioning.cattle.io/v1"
+  kindsRegexp: "."
+- apiVersion: "rke-machine-config.cattle.io/v1"
+  kindsRegexp: "."
+- apiVersion: "rke-machine.cattle.io/v1"
+  kindsRegexp: "."
+- apiVersion: "rke.cattle.io/v1"
+  kindsRegexp: "."
+- apiVersion: "apiextensions.k8s.io/v1"
```

Comment:
Suggesting to have rule for CRD to be the first one. It doesn't change anything functionally, but this way the configuration is a bit more logical and easier to read.